src/test/java/org/embulk/parser/json/TestJsonParserPlugin.java

```diff
@@ -37,12 +37,12 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
-import org.embulk.EmbulkTestRuntime;
 import org.embulk.config.ConfigSource;
 import org.embulk.spi.DataException;
 import org.embulk.spi.FileInput;
 import org.embulk.spi.Schema;
-import org.embulk.spi.TestPageBuilderReader.MockPageOutput;
+import org.embulk.test.EmbulkTestRuntime;
+import org.embulk.test.TestPageBuilderReader.MockPageOutput;
```

Comment:
Just note: [embulk 0.10.47](https://github.com/embulk/embulk/pull/1594) Move all `embulk-junit4` classes to `org.embulk.test`